console_backend/src/tabs/settings_tab.rs

```diff
@@ -277,27 +277,14 @@ impl SettingsTab {
     }
 
     fn import_success(&self) {
-        let mut builder = Builder::new_default();
-        let msg = builder.init_root::<crate::console_backend_capnp::message::Builder>();
```

Comment:
unrelated changes - was playing around with threads in settings and refactored this out into utils